### PR TITLE
Add file listing to VFS

### PR DESCRIPTION
**Pull request by westito**
For listing databases (ex. to delete them)

### CODE REVIEW

File: sqlite3/lib/src/wasm/file_system.dart [hunk 1]
```diff
@@ -35,6 +35,9 @@ abstract class FileSystem {
   /// otherwise.
   void deleteFile(String path);
 
+  /// List all files
+  List<String> listFiles();
```
Comment (simolus3) [inline sqlite3/lib/src/wasm/file_system.dart:39]: Let's expand this a bit:

```suggestion
  /// List all files.
  ///
  /// Returns a list of paths for files that currently exist in this file system.
  List<String> listFiles();
```

**Review by simolus3 — COMMENTED** [review]
Thanks for your contribution!

In b2aeb721f0a34c5ac7a51f89ce6ae7f24ac5efb5, I've added simple file system tests in `test/wasm/file_system_test.dart`. Could you rebase from there and add a simple test for `listFiles()` too? Something that initially matches an empty list, creates a bunch of files, checks again, deletes some files, checks again should be enough.

**Comment by simolus3** [comment]
The failing integration tests are due to https://github.com/subosito/flutter-action/issues/146, I'll run them manually and release this tomorrow. Thanks!
